models/iosxr/ntp_global/iosxr_ntp_global.yaml

```
@@ -0,0 +1,225 @@
+            aging_time:
+              type: int
+              description: Aging time in hours.
+            file:
+              description: File for drift values.
+              type: str
+        interface:
+          type: list
+          elements: dict
+          description: Configure NTP on an interface.
+          suboptions:
+            broadcast_client:
+              type: bool
+              description: Listen to NTP broadcasts
+            broadcast:
+              type: dict
+              description: Configure NTP broadcast service.
+              suboptions:
+                destination:
+                  type: str
+                  description: Configure broadcast destination address.
+                key:
+                  type: int
+                  description: Broadcast key number.
+                version:
+                  type: int
+                  description: <2-4>  NTP version number.
+            multicast:
+              key:
+                type: int
+                description: Configure multicast authentication key.
+              ttl:
+                type: int
+                description: Configure TTL to use.
+              client:
+                type: str
+                description:
+              destination:
+                type: str
+                description:
+              version:
+                type: int
+                description: <2-4>  NTP version number.
+        ipv4: &ip
+          description: Mark the dscp/precedence bit for ipv4 packets.
+          type: dict
+          suboptions:
+            dscp:
+              description: Set IP DSCP (DiffServ CodePoint).Please refer vendor document for valid entries.
+              type: str
+            precedence:
+              description: Set precedence Please refer vendor document for valid entries.
+              type: str
+              choices: [ "critical", "flash", "flash-override", "immediate", "internet", "network", "priority", "routine" ]
+        ipv6: *ip
+        log_internal_sync:
+          type: bool
+          description: Logs internal synchronization changes.
+        master:
+          type: int
+          description: Use NTP as clock source with stratum number <1-15>
+        max_associations:
+          type: int
+          description: <0-4294967295>  Number of associations.
+        passive:
+          type: bool
+          description: Enable the passive associations.
+        trusted_key:
+          type: 1
+          description: <1-65535>  Key number
+        update_calendar:
+          type: bool
+          description: Periodically update calendar with NTP time.
+        source:
+          type: str
+          description: Configure default interface.
+        servers:
+          description: Configure NTP server.
+          type: list
+          elements: dict
+          suboptions:
+            vrf: &vrf
+              description: vrf name.
+              type: str
+            host: &host
```

For consistency with how we have historically dealt with these attributes, can we call it `server`?
```suggestion
            server: &host
```

---

Following - neighbors -> neighbor.